BossMod/Autorotation/RPR/RPRActions.cs

```diff
@@ -44,25 +44,6 @@ public override void Dispose()
 
         public override Targeting SelectBetterTarget(AIHints.Enemy initial)
         {
-            // targeting for aoe
```

Comment:
Is it intended to be fully removed?

Reply:
reaper AOE is self targeted so this doesn't do anything. you can see that `NumTargetsHitByAOEGCD` doesn't take a parameter, so the for-loop here is performing the same redundant calculation for every nearby enemy. i think it must have just been copy-pasted from DRG and not updated